src/pages/admin/edit-event.tsx

```diff
@@ -0,0 +1,10 @@
+import EventForm from '~/components/Forms/Event'
+import RootLayout from '~/layouts/RootLayout'
+
+export default function EditEvent() {
```

Comment:
resolved the Admin Layout usage, but on my machine there is padding and the button never touched the footer. Tested in Firefox and Chrome. If you want I can add some in the parent flexbox of the form, but again on my machine this just adds padding. Doesn't matter if the form is in AdminLayout or RootLayout.

Reply:
I think this is because `AdminLayout` has enforced padding, while `RootLayout` does not.  Also please note my feedback about the form title needing to cange.

Reply:
Oh also, the `Callout` on this form should not be visible when editing.